ipaserver/install/cainstance.py

```diff
@@ -1304,7 +1304,14 @@ def __remove_lightweight_ca_key_retrieval_custodia(self):
         keyfile = os.path.join(paths.PKI_TOMCAT,
                                self.service_prefix + '.keys')
         keystore = IPAKEMKeys({'server_keys': keyfile})
-        keystore.remove_keys(self.service_prefix)
+        keystore.remove_server_keys_file()
+        try:
+            keystore.remove_keys(self.service_prefix)
```

Comment:
remove_keys() is internally calling remove_server_keys_file(), so we end up calling it twice in a row. It doesn't cause issue but can be simplified.